pkg/api/graph/graphview/dc_pipeline.go

```diff
@@ -12,8 +12,9 @@ import (
 type DeploymentConfigPipeline struct {
 	Deployment *deploygraph.DeploymentConfigNode
 
-	ActiveDeployment    *kubegraph.ReplicationControllerNode
-	InactiveDeployments []*kubegraph.ReplicationControllerNode
+	ActiveDeployment               *kubegraph.ReplicationControllerNode
+	ActiveDeploymentRestartingPods int32
```

Comment:
> @deads2k @smarterclayton bump? are you OK with caching the restarts in pipeline?

It doesn't bother me enough to stop it, but I think @kargakis is right about cleanliness.  A function that takes the graph and the RCNode seems cleaner.